### ISSUE TITLE
Garbage frame number reset waiting...

### ISSUE THREAD

**Issue by sunghyun-nam** [groupgets/pylepton#77]
Hello.

I have modified some code while keeping the l.capture() and Lepton3, Lepton files unchanged.

I sometimes encounter image corruption issues, as shown in the attached image, accompanied by the message "Garbage frame number reset waiting...".

From what I've read in other people's posts, it seems that the absence of the I2C interface in this project might be causing the image corruption issue. 

Is it really due to this problem that I'm experiencing the image corruption?

Thanks in advance.

![error](https://github.com/user-attachments/assets/84231708-76a5-448f-99a1-681c71674060)

**Comment by stevemessinger** [comment]
@sunghyun-nam Were you able to fix this issue? Getting this pretty often on my end as well

**Comment by MatteoGigerSR** [comment]
@stevemessinger Are you using a raspberry PI?

**Comment by stevemessinger** [comment]
@MatteoGigerSR yeah using a pi zero 2 w over spi. Going to try and flip to USB and see if that fixes the issue
